docs/storage.md

```diff
@@ -90,6 +90,7 @@ config:
     insecure_skip_verify: false
   trace:
     enable: false
+  prefix: ""
```

Comment:
I would be curious and actually I think it's must-have to actually make this change on bucket level, not each bucket client level.. WDYT? (: